parlai/tasks/convai2_wild_evaluation/build.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+# Download and build the data if it does not exist.
+
+import os
+import json
+import random
+
+import parlai.core.build_data as build_data
+
+
+def make_parlai_format(data: list, dpath: str):
+    train_p = 0.6
+    valid_p = 0.2
+    test_p = 1 - (train_p + valid_p)
+
+    assert train_p > 0
+    assert valid_p > 0
+    assert test_p > 0
+
+    random.seed()
```

Comment:
we don't want a random seed changing the dataset for everyone who downloads it here

Reply:
probably better to just take every other n lines or so?

Reply:
jaseweston, sorry for being absent - was on my vacation. Removed random seed from the dataset extraction.